addons/binding/org.openhab.binding.astro/README.md

```diff
@@ -162,3 +162,100 @@ then
+Number   ASTRO_MOON_ELEVATION            "Höhe"                                                     <moon>    { channel="astro:moon:home:position#elevation" }
+DateTime ASTRO_MOONECLIPSE_TOTAL         "Totale Finsternis[%1$td.%1$tm.%1$ty %1$tH:%1$tM]"         <moon>    { channel="astro:moon:home:eclipse#total" }
+DateTime ASTRO_MOONECLIPSE_PARTIAL       "Partielle Finsternis[%1$td.%1$tm.%1$ty %1$tH:%1$tM]"      <moon>    { channel="astro:moon:home:eclipse#partial" }
+Number   ASTRO_MOONPERIGEE_KM            "Perigäum[%.2f KM]"                                        <moon>    { channel="astro:moon:home:perigee#kilometer" }
+DateTime ASTRO_MOONPERIGEE_DATE          "Perigäum[%1$td.%1$tm.%1$ty %1$tH:%1$tM]"                  <moon>    { channel="astro:moon:home:perigee#date" }
+Number   ASTRO_MOONAPOGEE_KM             "Apogäum[%.2f KM]"                                         <moon>    { channel="astro:moon:home:apogee#kilometer" }
+DateTime ASTRO_MOONAPOGEE_DATE           "Apogäum[%1$td.%1$tm.%1$ty %1$tH:%1$tM]"                   <moon>    { channel="astro:moon:home:apogee#date" }
+
+//General
+String   ASTRO_DAYTIME                   "Tageszeit [%s]"                                           <sun>     { channel="astro:sun:home:phase#name" }
+String   ASTRO_ZODIAC                    "Sternzeichen[MAP(zodiac.map):%s]"                         <none>    { channel="astro:sun:home:zodiac#sign" }
+DateTime ASTRO_ZODIAC_START              "Von[%1$td.%1$tm.%1$ty]"                                   <none>    { channel="astro:sun:home:zodiac#start" }
+DateTime ASTRO_ZODIAC_END                "Bis[%1$td.%1$tm.%1$ty]"                                   <none>    { channel="astro:sun:home:zodiac#end" }
+String   ASTRO_SEASON                    "Jahreszeit[MAP(season.map):%s]"                           <none>    { channel="astro:sun:home:season#name" }
+```
+
+Sitemap:
+```
+Text label="Astronomische Daten" icon="sun" {
+    Frame label="Sonne" {
+        Text item=ASTRO_SUNRISE_TIME
+        Text item=ASTRO_SUNSET_TIME
+        Text item=ASTRO_AZIMUTH
+        Text item=ASTRO_ELEVATION
+        Text item=ASTRO_TOTAL_RADIATION
+        Text item=ASTRO_SUNECLIPSE_RING
+        Text item=ASTRO_SUNECLIPSE_PARTIAL
+        Text item=ASTRO_SUNECLIPSE_TOTAL
+    }
+    Frame label="Mond" {
+        Text item=ASTRO_MOONRISE_TIME
+        Text item=ASTRO_MOONSET_TIME
+        Text item=ASTRO_MOON_AZIMUTH
+        Text item=ASTRO_MOON_ELEVATION
+        Text item=ASTRO_MOONPERIGEE_DATE
+        Text item=ASTRO_MOONPERIGEE_KM
+        Text item=ASTRO_MOONAPOGEE_DATE
+        Text item=ASTRO_MOONAPOGEE_KM
+        Text item=ASTRO_MOONECLIPSE_PARTIAL
+        Text item=ASTRO_MOONECLIPSE_TOTAL
+        Text item=ASTRO_MOONPHASE
+    }
+    Frame label="Sonstiges" {
+        Text item=ASTRO_DAYTIME
+        Text item=ASTRO_ZODIAC
+        Text item=ASTRO_ZODIAC_START
+        Text item=ASTRO_ZODIAC_END
+        Text item=ASTRO_SEASON
+    }
+}
+```
+
+Transform season.map
```

Comment:
Hi,

Thanks for contributing and sharing! I would like to suggest the following. Add a caption 'Transformation definitons:' and below simply add 'season.map' and 'zodiac.map'